todo_app/data/trello_items.py

```diff
@@ -0,0 +1,89 @@
+import json
+import os
+import dotenv
+import requests
```

Comment:
I can't see the `requests` library in your `pyproject.toml` file, and when I run `poetry install` this isn't installed so the app errors here. We should make sure that's recorded for other devs by using `poetry add requests` and checking in `pyproject.toml` and `poetry.lock` changes, so that future devs have poetry know what libraries/versions to install

Reply:
(If you see the app working, then check whether you might be running with just `flask run`, in which case you're likely using your global environment rather than our virtual environment! We want to use `poetry run flask run`)